src/selectors/fridge.js

```diff
@@ -88,8 +87,16 @@ export const selectMinAndMaxDomains = createSelector(selectMinAndMaxLogs, minAnd
   };
 });
 
-export const selectBreaches = createSelector([selectSelectedFridge], fridge => {
-  const { breaches } = fridge ?? {};
-
-  return breaches;
-});
+export const selectBreaches = createSelector(
+  [selectTemperatureLogsFromDate, selectTemperatureLogsToDate, selectSelectedFridge],
+  (fromDate, toDate, fridge) => {
+    const { breaches } = fridge ?? {};
+    const breachesInDateRange = breaches.filtered(
+      'startTimestamp >= $0 && endTimestamp <= $1',
+      fromDate,
+      toDate
+    );
+
+    return breachesInDateRange;
```

Comment:
maybe not the best - only show a breach if it started within the range. Would be better to have this show a breach if it was anywhere within the range, but if it started before hand, adjust the breach values to show it beginning at the first x value in the chart. Could make an issue for this as it would take a little more work - just ensuring that the X value is at least as large as the smallest X value of the temperature logs (otherwise you get a funny chart with the breach icon stretching the X domain - i.e. breach starts on the 1st, where as temperature logs start on the 10th, funny chart